src/containers/SlideUpAccountDetails/index.tsx

```diff
@@ -28,7 +28,12 @@ const SlideUpAccountDetails: FC<ComponentProps> = ({account_number, close, githu
     <SlideUp className="SlideUpAccountDetails__SlideUp" close={close}>
       <div className="SlideUpAccountDetails__inner-wrapper">
         <div>
-          <img className="SlideUpAccountDetails__user-avatar" src={github_avatar_url} alt={github_username} />
+          <Avatar
+            alt={github_username}
+            className="SlideUpAccountDetails__user-avatar"
```

Comment:
@jamessspanggg If you have deleted the css, then there's no need of passing the class name.